db/query_test.go

```diff
@@ -268,9 +271,10 @@ func TestAccessQuery(t *testing.T) {
 	defer db.Close(ctx)
 	collection := GetSingleDatabaseCollectionWithUser(t, db)
 
-	db.ChannelMapper = channels.NewChannelMapper(`function(doc, oldDoc) {
+	_, err := collection.UpdateSyncFun(ctx, `function(doc, oldDoc) {
```

Comment:
UpdateSyncFun is going to write the sync function document to the database.  Setting the ChannelMapper (as the test was previously doing) is a bit more lightweight for test purposes - any reason we shouldn't just modify collection.ChannelMapper directly here?

Reply:
@torcolvin This question is still open - would like to know for future reference whether we can just continue to modify the ChannelMapper directly.

Reply:
Discussed offline, for future reference we can continue to take either approach, UpdateSyncFun or direct ChannelMapper updates.